### PR TITLE
fix: [CDS-81541]: Fixed page width of service page, spinner covering the scope tabs and added click on username in sidenav footer

### PR DESCRIPTION
**Pull request by Sagar-Rajput7**
### Summary

Fixes - 
 - PageSpinner blocking tabs in scope selector
 - Nested scroll in settings page
 - Service page width overflowing in both new and old nav versions
 - Added onClick on username in sidenav footer
 - Settings card bg color -> white
 - Home path redirection issue
 - CF - settings route destination breaking develop - Fix - added wrapper <></> around routeWithContext

#### Screenshots

https://github.com/harness/harness-core-ui/assets/98325173/9db8c03b-c8c8-43ce-a782-222d098d5812

https://github.com/harness/harness-core-ui/assets/98325173/5c1f2a0b-1ada-4757-863a-a4ca57cba24e

Screenshot 2023-10-16 at 10 07 08 AM

https://github.com/harness/harness-core-ui/assets/98325173/3f9e1c87-2152-4330-8f08-c0654ace0897

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

File: src/modules/10-common/navigation/SideNavV2/SideNavFooter/SideNavFooter.module.scss [hunk 1]
```diff
@@ -59,3 +59,7 @@
 .section {
   border-top: 1px solid var(--grey-200);
 }
+
+.cursor {
+  cursor: pointer;
```
Comment (MondalSayantan) [inline src/modules/10-common/navigation/SideNavV2/SideNavFooter/SideNavFooter.module.scss:64]: We can add this to profileLink class

**Review by MondalSayantan — DISMISSED** [review]

File: src/modules/10-common/navigation/SideNavV2/SideNavFooter/SideNavFooter.tsx [hunk 2]
```diff
@@ -105,7 +105,13 @@ const SideNavFooter: React.FC = () => {
   const { currentUserInfo: user } = useAppStore()
   const { getString } = useStrings()
   const { module } = getRouteParams<ModulePathParams>()
-  const match = useRouteMatch(routes.toUserProfile({ module: module }))
+  const history = useHistory()
+  const { params } = useGetSelectedScope()
+  const { pathname } = useLocation()
+  const matchRoute = matchPath<ModulePathParams>(pathname, {
```
Comment (abhinavrastogi-harness) [inline src/modules/10-common/navigation/SideNavV2/SideNavFooter/SideNavFooter.tsx:111]: do we need this? current mode and module should already be available from `useModuleInfo`

File: src/modules/75-cd/components/Services/ServicesContent/ServicesContent.module.scss [hunk 3]
```diff
@@ -17,7 +17,7 @@
   }
 
   .card {
-    --summary-widget-width: calc(100vw - 312px);
+    --summary-widget-width: calc(100vw - 338px);
```
Comment (abhinavrastogi-harness) [inline src/modules/75-cd/components/Services/ServicesContent/ServicesContent.module.scss:20]: why this change in this PR?

**Review by abhinavrastogi-harness — DISMISSED** [review]

File: src/modules/75-cd/components/Services/ServicesContent/ServicesContent.module.scss (hunk 3 above)
Comment (Sagar-Rajput7) [reply src/modules/75-cd/components/Services/ServicesContent/ServicesContent.module.scss]: [Update the code]

Here, the service page is breaking due to the right aux nav addition in both new and old nav version.

File: src/modules/10-common/navigation/SideNavV2/SideNavFooter/SideNavFooter.tsx (hunk 2 above)
Comment (Sagar-Rajput7) [reply src/modules/10-common/navigation/SideNavV2/SideNavFooter/SideNavFooter.tsx]: Updated, thanks for pointing out

**Review by avinashmadhwani02 — DISMISSED** [review]

**Review by avinashmadhwani02 — APPROVED** [review]

**Review by avinashmadhwani02 — DISMISSED** [review]

**Review by MondalSayantan — DISMISSED** [review]

File: src/modules/45-projects-orgs/components/ScopeSelector/OrgScopeSelector/OrgScopeSelector.tsx [hunk 4]
```diff
@@ -79,34 +79,36 @@ export const OrgScopeSelector = (props: OrgScopeSelectorProps): JSX.Element => {
...
+              itemCount={data?.data?.totalItems || 0}
+              pageSize={data?.data?.pageSize || 10}
+              pageCount={data?.data?.totalPages || 0}
+              pageIndex={data?.data?.pageIndex || 0}
+              gotoPage={pageNumber => setPage(pageNumber)}
+              hidePageNumbers
+            />
+          </Layout.Vertical>
+        ) : !loading ? (
+          <NoDataCard icon="nav-organization" message={getString('projectsOrgs.noOrganizations')} />
```
Comment (abhinavrastogi-harness) [inline src/modules/45-projects-orgs/components/ScopeSelector/OrgScopeSelector/OrgScopeSelector.tsx:109]: if you're using `Page.Body`, you don't need to render the `NoDataCard`. you can just pass the `noData` prop for it to render this instead.

**Review by abhinavrastogi-harness — DISMISSED** [review]

File: src/modules/45-projects-orgs/components/ScopeSelector/OrgScopeSelector/OrgScopeSelector.tsx (hunk 4 above)
Comment (Sagar-Rajput7) [reply src/modules/45-projects-orgs/components/ScopeSelector/OrgScopeSelector/OrgScopeSelector.tsx]: Thanks, forgot about update the noDataCard.

**Review by MondalSayantan — APPROVED** [review]

**Review by avinashmadhwani02 — APPROVED** [review]
